docs/user/tutorials/01-30-10-azure-vpc-peering.md

```diff
@@ -1,20 +1,23 @@
-# Create VPC Peering in Azure
+# Create Virtual Private Cloud Peering in Microsoft Azure
 
-This tutorial explains how to create a VPC peering connection between a remote VPC network and Kyma in Azure. The tutorial 
-assumes that the Cloud Manager module is enabled in your Kyma cluster. Follow the steps from this tutorial to create a new resource group, VPC 
-network and VM, and assign required roles to the provided Kyma service principal in your Azure subscription. If you want to
-use the existing resources instead of creating new ones, adjust variable names accordingly and skip
-the steps that create those resources.
+This tutorial explains how to create a Virtual Private Cloud (VPC) peering connection between a remote VPC network and Kyma in Microsoft Azure (Azure). Learn how to create a new resource group, VPC network and a virtual machine (VM), and assign required roles to the provided Kyma service principal in your Azure subscription. If you want to use the existing resources instead of creating new ones, adjust variable names accordingly and skip the steps that create those resources.
```

Comment:
```suggestion
This tutorial explains how to create a Virtual Private Cloud (VPC) peering connection between a remote VPC network and Kyma in Microsoft Azure (Azure). Learn how to create a new resource group, VPC network and a virtual machine (VM), and assign required roles to the provided Kyma service principal in your Azure subscription. If you want to use the existing resources instead of creating new ones, adjust variable names accordingly and skip the steps that create or delete those resources.
```

Should we put some message in the clean-up section as a heads up to the user not to delete existing resources.